# Message for the motor current measurements
float32[] motor_current